ISSUE_TEMPLATE.md

```diff
@@ -9,16 +9,22 @@
 The user story for the issue.
 
 ## What
-Specify in more detail what this issue adds to the repository.
+This issue demands 
+<!-- Specify in more detail what this issue adds to the repository. -->
 
 ## Why
-Specify in more detail why this issue is needed in the project.
+This issue is needed for the project because 
+<!-- Specify in more detail why this issue is needed in the project. -->
 
 ## How
-Specify in general terms how this feature can be added.
+This feature can be added by 
+<!-- Specify in general terms how this feature can be added. -->
 
 ## Definition of done
-Specify exactly what must be done before this task can be considered done.
+This task can be considered done when
+
+- 
+<!-- Specify exactly what must be done before this task can be considered done. -->
```

Comment:
@clenrock Because Github won't recognize it as a list otherwise, for some reason